src/Http/Http.Extensions/gen/Microsoft.AspNetCore.Http.RequestDelegateGenerator.csproj

```diff
@@ -17,6 +17,7 @@
 
   <ItemGroup>
     <InternalsVisibleTo Include="Microsoft.AspNetCore.Http.Extensions.Tests" />
+    <InternalsVisibleTo Include="Microsoft.AspNetCore.Http.Microbenchmarks" />
```

Comment:
Just curious: which internal types do we need this for? I had assumed that RunGenerator should just work by passing it a source and the generator.

Reply:
The `RequestDelegateCreationTestBase` class that I'm source-sharing between our integration tests and our benchmarks dependencies on some types that are internal to the generator (e.g. the `Endpoint` type which represents the info we've derived vai Roslyn about a method invocation that maps to a minimal API).

I could've done work work to factor out the RequestDelegateCreationTestBase class here but I figured there was no harm in doing IVT between our benchmarks and generator.

Also, it'll help in the future if we decide to profile specific parts of the generator implementation.

Reply:
I see, thanks for the explanation. I know other repos like dotnet/runtime try to avoid IVT for these cases and [prefer source sharing instead ](https://github.com/dotnet/runtime/blob/9fe580bf21f0b41e1a87b45c34ab6f2ff341dcd4/src/libraries/System.Text.RegularExpressions/tests/UnitTests/System.Text.RegularExpressions.Unit.Tests.csproj#L29-L47) but I think both approaches are fine 😃